### PR TITLE
Feature/weather app

### PR DESCRIPTION
**Pull request by kunmi02**
This PR is uses async and await to fetch data from the open weather API  and displays the weather info

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import getWeatherData from './modules/requester';
+import './styles.css';
+
+const displayValues = (response) => {
+  console.log(response);
```
Comment (uryelah) [inline src/index.js]: - [x] Please remove the console logs from your application, they are ok for debugging purposes but should be removed before submtiting for review or deploying the code as they make the code messy and overflow the browser terminal.

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+body {
+  color: white;
+  background-image: url("./assets/background.jpg");
```
Comment (uryelah) [inline src/styles.css:3]: - [ ] This background should be changing according to the response/weather in the submitted city.
You can change it simply by creating classes for different weathers and toggling them according to the API response.

File: src/modules/requester.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const baseUrl = 'http://api.openweathermap.org/data/2.5/weather?q=';
+const apiKey = '63f5550851a7e639b98f0c7c3e0e2e06';
+async function getWeatherData(city) {
+  const response = await fetch(`${baseUrl + city}&appid=${apiKey}`);
+  const data = await response.json();
+  return data;
+}
```
Comment (uryelah) [inline src/modules/requester.js]: - [x] Let's add error handling to your async functions with [try/catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch).

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Weather API</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="">
+  </head>
+  <body>
+    <h1>See your city's weather</h1>
```
Comment (uryelah) [inline dist/index.html:12]: - [x] Your application should also include a toggle button that defines if the temperature will be shown in Celcius or Fahrenheit, please add the toggle here and the necessary attached functions in the JS files..

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Azeez Azeez** , I'm Sarah your reviewer for this PR 👩‍🚀 

Good job starting out this project, you are working well with the API data and I specially like the error message you implemented in case the city entered is not valid.
There are still a couple of features you missed in this project so hang in there, you are almost there.

Keep on coding and Rock on! 🎸 👨‍🎤 

# Require Changes ♻️ 

- [ ] [OPTIONAL] Add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API.

- [ ] [OPTIONAL] Add a feature that fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location.

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 

**To highlight** 

- [ ] You have a descriptive Readme.
- [ ] Your Pull Request is nice. Good Job!
- [ ] You have deployed your website. Nice!!!
- [ ] You are using linters and no error is present. I am happy to see it. 
- [ ] Your project has met the minimum technical requirements too. 

In short, I am very excited to see this project and proud of you. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**
